(DOCSP-27109): Swift: Actor-Isolated Realm Concurrency Updates (#2731)
mongodb/docs-realm: source/sdk/swift/actor-isolated-realm.txt

.. _swift-actor-isolated-realm:

=================================
Actor-Isolated Realms - Swift SDK
=================================

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 2
   :class: singlecol

> Starting with Realm Swift SDK version [VERSION], Realm supports actor-isolated

Reviewer: To be updated before merge

realm instances. You can safely use an actor-isolated realm with Swift concurrency

> language features as an alternative to managing threading or dispatch queues.

Reviewer: ```suggestion
language features instead of managing threads or dispatch queues.
```
